### PR TITLE
GCP pages

### LINKED ISSUE #9397: GCP pages
Deliverables:

[ubuntu.com/gcp](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit#heading=h.c6wjbvvp8mcm)
[ubuntu.com/gcp/pro](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.n1426k6lmjxx)
[ubuntu.com/gcp/16.04](https://docs.google.com/document/d/14MeGfBfAiNxnKgJpTXhTEQpKMDAwoNp7Dqh0uqI7Dyc/edit)

These pages need to be no-index
These will be similar to ubuntu.com/azure and ubuntu.com/aws 

[Brief](https://docs.google.com/document/d/1bNYMeorz4TYqdJMLvrr_tyCUjdodQGwrmP1EC4IRQEA/edit)

### PR DESCRIPTION
**Pull request by carkod**
## Done

GCP pages:
[ubuntu.com/gcp](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit#heading=h.c6wjbvvp8mcm)
[ubuntu.com/gcp/pro](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.n1426k6lmjxx)

*Note*: [/16-04/gcp](https://docs.google.com/document/d/14MeGfBfAiNxnKgJpTXhTEQpKMDAwoNp7Dqh0uqI7Dyc/edit) not included, as I believe this is more relevant to include it in a /16-04 PR

## QA

Check these pages
- [ ] https://ubuntu-com-9398.demos.haus/gcp
- [ ] https://ubuntu-com-9398.demos.haus/gcp/pro

Against copy docs:
- [ubuntu.com/gcp](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit#heading=h.c6wjbvvp8mcm)
- [ubuntu.com/gcp/pro](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.n1426k6lmjxx)
Check links are correct
Check meta descriptions
Check copy doc links
Check no-index is present
Check form id = 3860

## Issue / Card

Fixes #9397

### CODE REVIEW

**Comment by lechsandecki** [comment]
thank you!

a few points that need be addressed:
1. 14.04 is not in the scope and shouldn't be listed 
2. the comparison table needs some love - can we have "launching soon" as a flag, rather than text that is making this whole sentence wrap around 2 lines, please? 
3. also added a link to 20.04 LTS marketplace listing on GCP side

**Comment by hugohuangtao** [comment]
Thank you!
Can we delete the "quote from Google" for now, which has been approved by Google yet? Once their PR approves it, we can add those words in. Thanks!

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:13]: This title does not match the copy doc

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:21]: Copy doc has "Ubuntu on Google Cloud"

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:42]: Copy doc has "UBUNTU ON Google Cloud POWERS CLOUD LEADERS LIKE"

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:98]: Copy doc has "LTS"

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:109]:
```suggestion
        <p>Run Ubuntu 18.04 LTS: Standard &nbsp;<span class="p-label--updated">Coming soon</span></p>
```

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:156]:
```suggestion
      <h2>Get Ubuntu Pro on GCP <span class="p-label--updated" style="position:relative;top: -1em;">Coming soon</span></h2>
```

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:12]: "Google Cloud"

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:14]: This is different in the copy doc

Comment (anthonydillon) [reply templates/gcp/pro.html:14]: Can you encode the dash with `&mdash;` please.

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:15]: Missing the js model revoke class

Comment (anthonydillon) [reply templates/gcp/pro.html:15]: Why does this have external class?

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:82]: This is different from the copy doc

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:74]: This is different from the copy doc

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:90]: This is different from the copy doc.

Comment (anthonydillon) [reply templates/gcp/pro.html:90]: Can you encode the dash with an `&mdash;` please

Comment (anthonydillon) [reply templates/gcp/pro.html:90]: Can you add a `&nbsp;` between the version and LTS please

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:111]: This mentions 14.04 when it shouldn't

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:122]: 30,000 now

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:127]: This is a draft quote, should be removed

File: templates/gcp/pro.html
Comment (anthonydillon) [inline templates/gcp/pro.html:112]: This should be a relative link

Comment (anthonydillon) [reply templates/gcp/pro.html:112]: Missing the js modal form class

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Few comments in the code

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: Done. I used the next `p-strip` but kept the `section` element.

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: Updated

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: So this keeps being updated, but then it makes things more inconsistent, because then there is the `Talk to us about Ubuntu on GCP` which again it's GCP instead of Google Cloud. So should I change all entries to Google Cloud? 
For now I updated it to reflect current copy doc.

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: Updated

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: This has been replaced by a link to the marketplace

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: Done. it doesn't seem to be vertically aligned, I changed it to  `-0.5em` instead

File: templates/gcp/pro.html
Comment (carkod) [reply templates/gcp/pro.html]: Updated

File: templates/gcp/pro.html
Comment (carkod) [reply templates/gcp/pro.html]: Yes this was mentioned by Lech.

File: templates/gcp/pro.html
Comment (carkod) [reply templates/gcp/pro.html]: Updated

File: templates/gcp/pro.html
Comment (carkod) [reply templates/gcp/pro.html]: Fixed

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: Done

File: templates/gcp/pro.html
Comment (carkod) [reply templates/gcp/pro.html]: Removed

File: templates/gcp/pro.html
Comment (carkod) [reply templates/gcp/pro.html]: All done

**Comment by anasereijo** [comment]
## /gcp

- [ ] Hero section: replace google logo with [svg version](https://assets.ubuntu.com/v1/6e176d9a-Google+Cloud+stacked.svg) and make it `width:250px`

- [ ] "Choose the right Ubuntu for you" section
I suggest changing making the cards like this: 
Screenshot 2021-03-17 at 10 20 11

- [ ] "Features" section: remove `p-divider__block`. so it will look like this:
Screenshot 2021-03-17 at 10 26 02

"Get Ubuntu Pro on GCP" section:
- [ ] remove the colon from the title
- [ ] move the "coming soon" label above the title and remove `top: -0.5em;`
- [ ] make 'read more' part of the link
Screenshot 2021-03-17 at 10 31 33


any questions let me know!

**Comment by anasereijo** [comment]
## /gcp/pro

Hero section: 
- [ ] replace google logo with [svg version](https://assets.ubuntu.com/v1/6e176d9a-Google+Cloud+stacked.svg) and make it `width:250px`
- [ ] get in touch should be a button

- [ ] "Broadest open source security coverage" section: make title 8 cols
Screenshot 2021-03-17 at 10 35 01

Table
- [ ] looks a bit squashed, please make it 12 cols
- [ ] Make Ubuntu and Ubuntu Pro headings `class="u-align--center"`
- [ ] Make "5 years" and "10 years"   `class="u-align--center"`





Screenshot 2021-03-17 at 10 39 19

**Comment by anasereijo** [comment]
Thanks for making these changes, @carkod 👍

File: templates/gcp/index.html
Comment (anthonydillon) [reply templates/gcp/index.html]: I thought we would want this as a superscript style so high but see what design think.

File: navigation.yaml
Comment (anthonydillon) [inline navigation.yaml:1518]: You are missing an overview I believe as so there is no second-level navigation.

File: templates/gcp/index.html
Comment (anthonydillon) [inline templates/gcp/index.html:142]: Should be an external link

**Review by anthonydillon — APPROVED** [review]
LGTM with some small comments in the code :+1:

File: navigation.yaml
Comment (carkod) [reply navigation.yaml:1521]: Fixed

File: templates/gcp/index.html
Comment (carkod) [reply templates/gcp/index.html]: Added class

**Pull request merged**
